src/main/java/de/teamlapen/vampirism/core/ModEventHandler.java

```diff
@@ -125,6 +133,12 @@ public void onPlayerLoggedIn(PlayerEvent.PlayerLoggedInEvent event) {
         }
         VampirismMod.dispatcher.sendTo(new SkillTreePacket(VampirismMod.proxy.getSkillTree(false).getCopy()), (ServerPlayerEntity) event.getPlayer());
 
+        Map<ResourceLocation, Pair<Map<ResourceLocation, Integer>, Integer>> bloodValues = Maps.newConcurrentMap();
+        bloodValues.put(new ResourceLocation(REFERENCE.MODID, "entities"), new Pair<>(((VampirismEntityRegistry) VampirismAPI.entityRegistry()).getBloodValues(), ((VampirismEntityRegistry) VampirismAPI.entityRegistry()).getBloodMultiplier()));
```

Comment:
Do we need a resource location with mod id here? Blood values added by other mods should be synchronized by Vampirism anyway so there should be no need for addons to use this or the mod id every being different